Documentation for logprob module (#6858)
pymc-devs/pymc: docs/source/api/order_stats.rst

================

> Order_Statistics

Reviewer: ```suggestion
Order Statistics
```

================

------------
Introduction
------------

> Order statistics is a way to understand and analyze data by looking at its ordered values. Imagine you have a set of numbers, and you want to know things like the smallest number, the largest number, or the middle value. The maximum, minimum and median respectively represent them. Order statistics help you find these values and more.

Reviewer: ```suggestion
Order statistics are statistics that look at the sorted values of a set of numbers. For instance, you want to perform inference on (i.e. compute the log-probability of observing) the smallest number, the largest number, or the middle value of a set of numbers.
```

> In this section, we'll tackle how users can find the log-probability corresponding to the nth order statistic (maximum value) using PyMC for their own Custom distributions.

Reviewer: ```suggestion
In this section, we'll tackle how users can find the log-probability corresponding to the maximum statistics (or nth order statistics) using PyMC.
```

> Example 1: [4, 2, 6]
> 	Minimum: The smallest number is 2.
> 	Median: Since there are three elements, the median is the middle value, which is 4.
> 	Maximum: The largest number is 6.
> 
> Example 2: [10, 3, 8]
> 	Minimum: The smallest number is 3.
> 	Median: Since there are three elements, the median is the middle value, which is 8.
> 	Maximum: The largest number is 10.
> 
> Example 3: [-1, 5, 3]
> 	Minimum: The smallest number is -1.
> 	Median: Since there are three elements, the median is the middle value, which is 3.
> 	Maximum: The largest number is 5.

Reviewer: ```suggestion
```

Reviewer: I would maybe remove this example, or rephrase it differently... We are interested in the log-prob inference of order statistics. Here, they are just numbers, not statistics

------------------------
`Max`
------------------------

> Using PyMC and Pytensor, users can extract the maximum of a distribution and derive the log-probablity corresponding to this operation.

Reviewer: ```suggestion
Using PyMC and Pytensor, users can extract the distribution of the maximum statistics and derive its log-probablity corresponding.
```
